.gitignore

```diff
@@ -0,0 +1,2 @@
+test.md
```

Comment:
[OPTIONAL]

- You did well adding the test.md file into the .gitignore folder 👍🏼 , but the file is still present in your repository which is because you committed the file before adding it to the .gitignore file. Kindly ensure that you add all necessary files to .gitignore before committing them 😄.

Reply:
Hi, @vickymarz Thanks a lot for bringing my attention to that. I will make sure to adhere to that next time.